### DISCUSSION TITLE
Cannot find the plugin in marketplace

### DISCUSSION THREAD

**Discussion by SHAY2407** [nevarok/NevarokML#4] [Q&A] [answered]
I cannot find the plugin in the marketplace, is it released yet ?

**Comment by nevarok** [comment, marked as answer]
> I cannot find the plugin in the marketplace, is it released yet ?

Hello @SHAY2407 
Currently, the plugin is not yet available in the marketplace as it is still in development. I am actively working on adding more features, examples, and running tests to ensure the plugin's stability before its release.

The latest version of the plugin is hosted on GitHub, and you can access it there. I will keep you informed here as soon as it becomes available in the marketplace.

If you encounter any issues or need assistance with installation or usage, please don't hesitate to reach out to me. Your feedback is also greatly appreciated.

Thank you for your patience and support.

**Reply by SHAY2407** [reply]
Thanks for the response 👍, i actually don't know how to build a library, i downloaded the repo and pasted the plugin in the marketplace folder in plugins, it told me that 3 files are need to be built from the IDE, that is why I am waiting for the marketplace version,

**Reply by nevarok** [reply]
I just pushed binaries and intermediate folders. Could you please try and pull the repository again to check if it solves your issue?

**Reply by SHAY2407** [reply]
After applying the plugin from the edit option inside unreal, i restart it, first it tells me that the plugin was built for 5.2, i click on load it anyway, then it gives me this message:

The following modules are missing or built with a different engine version:

  NevarokML
  NevarokMLEditor
  NevarokMLNodes

Engine modules cannot be compiled at runtime. Please build through your IDE.

**Reply by nevarok** [reply]
Hello @SHAY2407 

Please ensure that you are using Unreal Engine version 5.2, as the plugin is currently only supported for this specific version.

It's worth noting that there were significant API changes within the Unreal Engine NNE Plugin for version 5.3, which necessitated some restructuring.

Rest assured, I am actively working on providing support for Unreal Engine 5.3 and will keep you updated on the progress. Thank you for your patience and understanding!

**Reply by SHAY2407** [reply]
it's giving the same error on 5.2

**Reply by nevarok** [reply]
Hello @SHAY2407 ,

I sincerely appreciate your patience and your interest in using our plugin. I understand that you're experiencing some difficulties, and I'm here to assist you in resolving them.

I wanted to let you know that I have identified a potential issue with the repository. It appears that some files were mistakenly ignored and were missing from the repo. I have taken immediate action to rectify this by updating the .gitignore file and uploading the missing files.

Could you please try pulling the repository again and see if this resolves the issue you were facing? Your feedback on this matter is invaluable, and I'm committed to ensuring that our plugin functions smoothly for you.

Once again, thank you for your understanding and for bringing this matter to my attention. Please don't hesitate to reach out if you encounter any further issues or have additional questions. Your support is greatly appreciated.

**Reply by nevarok** [reply]
@SHAY2407 Additionally, I have created a [video tutorial](https://www.youtube.com/watch?v=nFbgVGCifA8) that outlines the installation process directly from our GitHub repository. This tutorial should provide you with a clear guide to successfully set up the NevarokML plugin in Unreal Engine 5.2.

Thank you once again for your patience. I look forward to assisting you further and ensuring a smooth experience.

**Reply by SHAY2407** [reply]
thank you so much, it is now working perfectly!

**Reply by nevarok** [reply]
@SHAY2407  I'm delighted to hear that everything is now working perfectly for you. Should you encounter any future issues or require assistance, please do not hesitate to reach out. Your continued interest and support are greatly appreciated!
